src/algopy_testing/op.py

```diff
@@ -847,6 +847,62 @@ def app_opted_in(
     return app_id in account_data.opted_apps
 
 
+class _AcctParamsGet:
+    def __getattr__(
+        self, name: str
+    ) -> typing.Callable[[algopy.Account | algopy.UInt64 | int], tuple[typing.Any, bool]]:
+        def get_account_param(a: algopy.Account | algopy.UInt64 | int) -> tuple[typing.Any, bool]:
+            import algopy
+
+            from algopy_testing.context import get_test_context
+
+            context = get_test_context()
```

Comment:
this check occurs in a lot of methods, might be worth having another variant that does this check and returns a non None context

Reply:
Resolved, feel free to reopen if you think something else is missing